source/fundamentals/network-compression.txt

```diff
@@ -0,0 +1,97 @@
+.. _golang-network-compression:
+
+===================
+Network Compression
+===================
+
+In this guide, you can learn how to enable **network compression** by using
+the {+driver-short+}. You can specify a driver option to compress messages,
+which reduces the amount of data passed over the network between MongoDB and
+your application.
+
+The {+driver-short+} supports the following compression algorithms:
+
+1. `Snappy <https://google.github.io/snappy/>`__: available in MongoDB 3.4 and later.
+
+2. `Zlib <https://zlib.net/>`__: available in MongoDB 3.6 and later.
+
+3. `Zstandard <https://github.com/facebook/zstd/>`__: available in MongoDB 4.2 and later.
+
+If you specify multiple compression algorithms, the driver selects the
+first one in the list supported by your MongoDB instance.
+
+.. note::
+
+   When using the Snappy or Zstandard compression algorithm, you must
+   :ref:`add explicit dependencies <compression-dependencies>`.
+
+.. _enable-compression:
+
+Specify Compression Algorithms
+------------------------------
+
+You can enable compression for the connection to your MongoDB instance
+by specifying the algorithms in one of two ways: 
+
+1. Adding the ``compressors`` parameter to your connection string. 
+2. Using the ``SetCompressors()`` method to specify a ``ClientOptions`` instance.
+
+.. tabs::
+
+   .. tab:: Connection String
+      :tabid: connection-string
+
+      To enable compression using the connection string, add the
+      ``compressors`` parameter to your connection string. You can
+      specify one or more compression algorithms separated by commas:
+
+      .. code-block:: go
+         :emphasize-lines: 1
+
+            opts := options.Client().ApplyURI("mongodb://localhost:27017/?compressors=snappy,zlib,zstd")
+            client, _ := mongo.Connect(context.TODO(), opts)
+
+   .. tab:: SetCompressors()
+      :tabid: mongoclientoptions
+
+      To enable compression using `SetCompressors()
+      <{+api+}/mongo/options#ClientOptions.SetCompressors>`__, pass
+      the compression algorithm you want to use to the ``SetCompressors()``
+      method. You can specify one or more compression algorithms separated
+      by commas:
```

Comment:
Issue:
I think this is missing some important information. In the code example, it shows that they must be elements of a string array, which happens to be defined inline by using commas, but could've been added in many other ways. 

Suggestion:
I think it would be more accurate to mention the type that SetCompressors() accepts (appears to be a string array) than to mention that they must be separated by commas.